knowledge-test.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"strconv"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func generateTestFlashcards(fc flashcard, testRange int) flashcard {
```

Comment:
Dodałbym jakiś komentarz jak to ma działać, żeby było wiadomo, czego się spodziewać.

Przy `len(fc) == 4` zwrócona mapa ma rozmiar 4 przy `testRange` równym 3, 4 i 5 (i więcej ofc). Wydaje mi się, że przy `testRange == 3` funkcja powinna zwracać 3 fiszki.